Applications/MmrMobile/Contracts/AlertEvent.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+
+namespace HomagConnect.MmrMobile.Contracts
+{
+    public class AlertEvent : IExtensibleDataObject
+    {
+        /// <summary>
+        /// Timestamp, when the Event started
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public DateTimeOffset EventStartTime { get; set; }
+
+        /// <summary>
+        /// Timestmp for the end of the event
+        /// If the event is still active, the current timestamp is returned
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public DateTimeOffset EventEndTime { get; set; }
+
+        /// <summary>
+        /// Unique identifier for one event
+        /// </summary>
+        [JsonProperty(Order = 2)]
+        public Guid EventInstanceId { get; set; }
+
+        /// <summary>
+        /// Machine-Number, Format "x-yyy-zz-nnnn"
+        /// </summary>
+        [JsonProperty(Order = 3)]
+        public string? MachineNumber { get; set; }
+
+        /// <summary>
+        /// value = 1 - 1000
+        /// </summary>
+        [JsonProperty(Order = 4)]
+        public short Severity { get; set; }
```

Comment:
Can we introduce for the severity an enumeration?

Reply:
Enum from 1 to 1000? 
With which names for each value? English the number?

Reply:
Ok, then leave it as it is.